LoggingKata/Program.cs

```diff
@@ -12,18 +12,36 @@ class Program
 
         static void Main(string[] args)
         {
-            logger.LogInfo("Log initialized");
-
             var lines = File.ReadAllLines(csvPath);
-
-            logger.LogInfo($"Lines: {lines[0]}");
-
             var parser = new TacoParser();
-
             var locations = lines.Select(parser.Parse).ToArray();
 
-            // TODO:  Find the two Taco Bells in Alabama that are the furthest from one another.
-            // HINT:  You'll need two nested forloops
+            logger.LogInfo("LogInfo");
+
+            ITrackable TacoBell1 = null;
+            ITrackable TacoBell2 = null;
+            double distance = 0;
+            for (int i = 0; i < locations.Length; i++)
+            {
+                logger.LogInfo($"Lines: {lines[i]}");
+                for (int k = 0; k < i + 1; k++)
+                {
+                    ITrackable LocationA = locations[i];
+                    var corA = new GeoCoordinate(LocationA.Location.Latitude, LocationA.Location.Longitude);
```

Comment:
These lines 29-30 could be outside of this loop (k) but still inside the outer loop (i), such as:
```CS
for (int i = 0; i < location.Length; i++)
{
  ITrackable LocationA = locations[i];
  var corA = new GeoCoordinate(LocationA.Location.Latitude, LocationA.Location.Longitude);
  
  for (int k...
```